network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -55,7 +57,7 @@ def get_product_subset(cutoff_date, authenticated, key, products, language_code=
     """
     products = products.filter(
         review_date__gte=cutoff_date,
-        locale=Locale.objects.get(language_code=language_code)
+        # locale=Locale.objects.get(language_code=language_code)
```

Comment:
we don't actually want to filter for locale atm, because it's exclusive filtering, and would strip all EN products on non-EN locales